include/klee/Interpreter.h

```diff
@@ -38,7 +38,9 @@ class InterpreterHandler {
   virtual llvm::raw_ostream &getInfoStream() const = 0;
 
   virtual std::string getOutputFilename(const std::string &filename) = 0;
-  virtual llvm::raw_fd_ostream *openOutputFile(const std::string &filename) = 0;
+
+  virtual std::unique_ptr<llvm::raw_fd_ostream>
```

Comment:
If we are changing this, I think API that returns the actual file instead of `llvm::raw_fd_ostream` is more useful in general. Not sure how annoying that is in the rest of the code.